### ISSUE TITLE
"Requests to the server have been blocked by an extension" boxes.

### ISSUE THREAD

**Issue by Werwa** [EFForg/privacybadgerchrome#907]
![ss 2016-08-31 at 01 28 54](https://cloud.githubusercontent.com/assets/7074347/18139324/f0df485c-6f7e-11e6-9be3-ec302256f987.png)
I noticed on some pages the ad is just replaced with a grey box saying "Requests to the server have been blocked by an extension". That's good and all since it's actually blocking the ad, but now instead of having a bunch of ads on the webpage there's a bunch of grey boxes, is there a way for the boxes to be removed along with the ad? Adblock Plus seems to remove both the ads and the grey box, and when I switched from using ABP to Privacy Badger, everything seems to be more cluttered with boxes.
I know this isn't really a bug, but I couldn't find anywhere else to really post this.

Sorry if this isn't the right place to post this, or if I did something wrong. This is pretty much the first time I really did anything on this site so I'm not very familiar on how to use it yet.

**Comment by cowlicks** [comment]
@Werwa just FYI about "switching" from ABP to privacy badger. You can use both. Privacy Badger doesn't do everything ABP does and vice versa.

However we should collapse divs better if we block their content. Related to #442

**Comment by Werwa** [comment]
@cowlicks Oh, yeah I know I probably _can_ use both, but the main reason I decided to use Privacy Badger over ABP was because of the fact that it allows ads to be shown if they respect the "do not track" thing (a pretty neat idea!), ABP doesn't really do that. Besides, I think it'd be kind of redundant if you kept an extension that blocks ads, when you just got an extension that not only blocks most ads, but also does a bunch more!

**Comment by cooperq** [comment]
Yea I would be happy to have this feature, but it is not at the top of my priority list. But someone else is free to run with it and I would happliy accept a patch!
Closing this as a duplicate of #442
